app/presenters/coronavirus/sub_section_json_presenter.rb

```diff
@@ -29,70 +14,27 @@ def output
     @output ||=
       {
         title: title,
-        sub_sections: sub_sections,
+        sub_sections: create_sub_sections,
```

Comment:
Minor: I don't think you needed to rename the method.

Reply:
undone, thanks